src/docs/app-monitoring/user-defined-monitoring.md

```diff
@@ -194,6 +194,27 @@ oc apply -f app-alerting-rule.yaml
 
 `AlertmanagerConfig` is part of the Prometheus Operator, which helps manage Prometheus instances and configurations in Kubernetes. Using AlertmanagerConfig ,allows you to manage and version control your alerting rules and configurations within Kubernetes resources, keeping them aligned with your application code and infrastructure. More information about [Alertmanager](https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/docs/platform-automation/alertmanager/)
 
+Please have this network policy added to ensure that proper metrics are scraped
+
+```yaml
+  apiVersion: networking.k8s.io/v1
+  kind: NetworkPolicy
+  metadata:
+    name: allow-from-openshift-monitoring
+    namespace: be1c6b-test
+  spec:
+    ingress:
+      - from:
+          - namespaceSelector:
+              matchLabels:
+                kubernetes.io/metadata.name: openshift-user-workload-monitoring
+    podSelector: {}
+    policyTypes:
+      - Ingress
+```
+
+Here we can see that label under `service` should match with the `ServiceMonitor` under selector and matchLabels.
```

Comment:
I don't think we need this sentence now that the service and servicemonitor yaml was removed? Or maybe it should be higher up in the doc where those objects are described.

Reply:
removed